README.md

```diff
@@ -1,86 +1,56 @@
-[Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
-===========================================================
+# [Cozy](https://cozy.io) Drive for Desktop <sup>(beta)</sup>
+
+[![Cozy Logo](https://raw.github.com/cozy/cozy-setup/gh-pages/assets/images/happycloud.png)](https://cozy.io)
 
 [![Latest release](https://img.shields.io/github/release/cozy-labs/cozy-desktop/all.svg)](https://github.com/cozy-labs/cozy-desktop/releases)
 [![Licence](https://img.shields.io/github/license/cozy/cozy-drive.svg)](https://github.com/cozy/cozy-drive/blob/master/LICENSE)
 <br>
 [![Windows build](https://img.shields.io/appveyor/ci/cozy/cozy-desktop/master.svg?label=windows)](https://ci.appveyor.com/project/cozy/cozy-desktop/branch/master)
-[![Linux & macOS build]( 	https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
+[![Linux & macOS build](https://img.shields.io/travis/cozy-labs/cozy-desktop/master.svg?label=mac/linux)](https://travis-ci.org/cozy-labs/cozy-desktop/branches)
 [![Code climate](https://img.shields.io/codeclimate/github/cozy-labs/cozy-desktop.svg)](https://codeclimate.com/github/cozy-labs/cozy-desktop)
 [![Coverage](https://img.shields.io/codecov/c/github/cozy-labs/cozy-desktop/master.svg?label=coverage)](https://codecov.io/gh/cozy-labs/cozy-desktop/list/master)
 
-## What is Cozy?
+Cozy Drive for Desktop allows you to synchronize the files stored in [your Cozy](https://cozy.io) with your laptop and/or desktop computer.
+It replicates your files on your hard drive and applies the changes you make to them on other synchronized devices and on [your online Cozy](https://github.com/cozy/cozy-stack).
+
+You can download it for Windows, macOS, and Linux on [**Cozy Drive for Desktop download page**](https://cozy.io/en/download/).
```

Comment:
Github is not an entry point for users that just want to download the application. I don't think a big fat download text is required.

# [Download](https://cozy.io/en/download/)

This is discussed [here](https://github.com/cozy-labs/cozy-desktop/issues/1141#issuecomment-417234860) by @sebn. Let me know if it is a strong requirement or just a thought.

Nothing is written on marble, we could change it later if needed.

Reply:
I still believe the link name is not so clear.
But this is definitely not a requirement.